src/site/content/en/progressive-web-apps/promote-install/index.md

```diff
@@ -83,37 +94,62 @@ who aren't interested in installation.
 
 <div class="w-clearfix">&nbsp;</div>
 
-### Fixed header {: #header }
+### Simple install button {: #simple-button }
 
-<figure class="w-figure w-figure--inline-right">
-  <img src="header.png" alt="Custom install button in header">
+The simplest possible UX is to include an 'Install' or 'Get app' button at an
+appropriate location in your web content. Ensure the button doesn't block
+other important functionality and is out of the way of the user's flow in
+the user's journey for your application.
```

Comment:
@petele does this text have some extra words, or is this written as intended: `out of the way of the user's flow in the user's journey`

Reply:
I feel this para might do with a re-write, it feels a little clunky. Basically what we are saying is a simple install button is an easy way to engagement users just make sure it is placed in the UI where is can be seen but doesnt block the user from doing anything else, for example on a shopping site is doesnt block a user from adding something to their cart or reading an article on a news site etc.

Reply:
```suggestion
other important functionality and is out of the way of the user's journey
through your application.
```
I think this is good enough